### PR TITLE
Add OWNERS file

### PR DESCRIPTION
**Pull request by kashifest**
Add the OWNERS file

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+approvers:
+ - lentzi90
```
[the remark this answers is not in the document]
Comment (kashifest) [reply OWNERS:4]: Ow, I sorted this here https://github.com/metal3-io/community/pull/1/files#diff-c393dd2b18a8a597d895940893e574d6895122c4bf3ce6e2a78dbd9ca0cdd66f , and took the file from there, strange!

[the remark this answers is not in the document]
Comment (kashifest) [reply OWNERS:4]: Lets take it in, and the in the other PR it will be sorted

[the remark this answers is not in the document]
Comment (tuminoid) [reply OWNERS:4]: Sure.

**Review by tuminoid — APPROVED** [review]

**Review by Rozzii — APPROVED** [review]
/lgtm

**Pull request merged**